### DISCUSSION TITLE
Extra sticky filter for file picker

### DISCUSSION THREAD

**Discussion by manuel2258** [helix-editor/helix#7912] [Ideas] [answered]
Hey there,

I really enjoy using helix as my daily driver. 
However I often found myself in situations where I would have liked to apply a 'sticky' filter for the file picker, which is kept when exiting the file picker (not between multiple helix sessions). This would be useful when for example wanting to inspect multiple files in a single module, or only wanting to see all .cpp files, etc ...

Implementation wise I would have though about a few new commands which could be used to set and delete the file picker context.

Would you guys be interested in the feature / would merge it upstream? 
Any more ideas or suggestions towards it? 
I would like to do the implementation myself.

**Comment by the-mikedavis** [comment, marked as answer]
You can use Alt+Enter in a picker to open the selection in the background without closing the picker or `<space>'` to bring up the last picker that was open. Extra commands that control state in pickers seems like unnecessary complexity to me.

**Reply by manuel2258** [reply]
Wow I didn't knew about 'Alt+Enter' and '<space>'', they fully solve my usecase and therefore I fully agree that any extra implementation would just be really unnecessary.
Thanks for the reply and the good work!
